docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,194 @@
+# Convert Data Operation
+The ```$convert-data``` operation allows data conversion from legacy formats (currently supports **Hl7v2** only) to FHIR format. This feature is currently turned off by default. To enable the feature, update the `FhirServer:Operations:ConvertData:Enabled` setting to be true.
+
+## Conversion Template Collection
+The convert data operation is based on the [FHIR Converter Project](https://github.com/microsoft/FHIR-Converter/tree/dotliquid).
+To make a data conversion call, you need a template collection to define the mappings between different data formats.
+
+We have a built-in template collection (hl7v2 to FHIR R4) which is a copy from the FHIR Converter project.
+You can directly use the built-in template collection reference ```microsofthealth/fhirconverter:default``` in the request payload.
```

Comment:
Here we should clarify when will these default templates be refreshed on the server? Also, how can a user know what version of the template is being used as the default template? This is important because if the default templates change, the conversion behavior will change, affecting the pipeline.

Reply:
Added the template version clarification.